src/main/lrsql/init/git_data.clj

```diff
@@ -0,0 +1,30 @@
+(ns lrsql.init.git-data
+  (:require [cheshire.core :as json]
+            [clojure.string]))
+
+(defn read-git []
+  (let [read-text (or (try (slurp "target/bundle/git-details.json")
+                           (catch Exception _
+                             nil))
+                      (try (slurp "dev-resources/git-details.json")
+                           (catch Exception _
+                             nil)))]
```

Comment:
The production path here won't work and the dev one won't unless you do a manual copy, I think we need to work on this bit.

Reply:
what path would work here for prod?  This is a jvm thing I'm afraid I don't understand.

Reply:
Let's go over it on slack

Reply:
:+1: